.azure/pipelines/quarantined-pr.yml

```diff
@@ -86,9 +86,6 @@ jobs:
     steps:
     - powershell: "& ./eng/build.ps1 -CI -nobl -all -pack -NoBuildJava"
       displayName: Build
-    # The templates part can be removed when the Blazor Templates run on Helix
```

Comment:
Is there a reason the Mac and Linux legs weren't given the same treatment?

Reply:
Oops, I didn't even realize we ran those on mac/linux, I will remove those as well.

Reply:
https://github.com/dotnet/aspnetcore/pull/33002